src/components/Button/index.vue

```diff
@@ -8,6 +8,9 @@
     :style="dark ? 'color:white' : ''"
     :disabled="disabled"
     :loading="loading"
+    v-on="on"
+    v-bind="bind"
```

Comment:
why are you using these props? vue already provides $listeners and $attrs please update it instead of you using props for this one